<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MacBook-Pro.local" failures="2" tests="2" name="Error Validation" time="30.499" errors="0" timestamp="2024-07-10T20:32:02 IST">
  <testcase classname="practiceProject.frameworkDesign.errorValidation" name="errorValid" time="0.906">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;Incorrect email password.&amp;gt; but was:&amp;lt;Incorrect email or password.&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<Incorrect email password.> but was:<Incorrect email or password.>
at practiceProject.frameworkDesign.errorValidation.errorValid(errorValidation.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- errorValid -->
  <testcase classname="practiceProject.frameworkDesign.errorValidation" name="producValid" time="23.531">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at practiceProject.frameworkDesign.errorValidation.producValid(errorValidation.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- producValid -->
</testsuite> <!-- Error Validation -->
